deploy/aws/aws-tutorial.tfvars

```diff
@@ -0,0 +1,11 @@
+pd_instance_type = "c5d.large"
+tikv_instance_type = "c5d.large"
+tidb_instance_type = "c5d.large"
+monitor_instance_type = "c5d.large"
```

Comment:
Local SSD of monitor and tidb instance is not used, so I suppose `c5.large` would be better?

Reply:
This is a good point, default for TiDB is actually a `c4.4xlarge` so maybe just `c4.large` for TiDB and `c5.large` for monitor?